test/unit/spec/v2/view-builder/internals/FormInputFactory_spec.js

```diff
@@ -516,6 +516,25 @@ describe('v2/view-builder/internals/FormInputFactory', function() {
           },
           authenticatorKey: 'okta_verify'
         },
+        {
+          label: 'Google Authenticator',
```

Comment:
can we also add google authenticator to `handles authenticatorVerify Select type`